dev/acf-export.json

```diff
@@ -0,0 +1,569 @@
+[
+    {
+        "key": "group_5c9cddeea9e45",
+        "title": "EDU News Feed Options",
+        "fields": [
+            {
+                "key": "field_5c9cddf6a827b",
+                "label": "Stories",
+                "name": "main_site_stories",
+                "type": "relationship",
+                "instructions": "Add stories to be displayed on www.ucf.edu.",
+                "required": 1,
+                "conditional_logic": 0,
+                "wrapper": {
+                    "width": "",
+                    "class": "",
+                    "id": ""
+                },
+                "post_type": [
+                    "post"
+                ],
+                "taxonomy": "",
+                "filters": [
```

Comment:
I'd suggest removing the by-taxonomy filtering in a future update...the dropdown that gets generated is very slow to load and makes the interface look a bit overwhelming at first glance.